services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/utils/MongoDbServiceUtils.java

```diff
@@ -33,7 +33,8 @@
     /**
      * Checks if a provided error is {@link MongoException} with error category {@link ErrorCategory#DUPLICATE_KEY}.
      *
-     * @param throwable The exception to check. Other Throwables but {@link MongoException} return {@code false}.
+     * @param throwable The exception to check. Other Throwables but {@link MongoException} return {@code
```

Comment:
Could you please move the `@code` part to the next line and  this change is also applicable to t other javadocs.

Reply:
done